content/introduction/getting-started.md

```diff
@@ -112,33 +112,33 @@ And... bask in the glory of your first Hanami project at
 ## Hanami's Architecture
 
 Hanami's architecture revolves around your project containing many `apps`.
-These all live together in the same codebase, and are run in the same Ruby process.
+These all live together in the same codebase, and exist in the same Ruby process.
 
 They live under `apps/`.
 
-By default, we have a `web` app, which can be thought of as the normal, user-facing web interface.
+By default, we have a `web` app, which can be thought of as the standard, user-facing web interface.
 This is the most popular, so you'll probably want to keep it in your future Hanami projects.
-But, just so you know, there's nothing special or different about this app, it's just so common that Hanami generates it for us.
+However, there's nothing unique about this app, it's just so common that Hanami generates it for us.
 
-Later (in a real project), we would add other apps, such as an `admin` panel, a JSON `api`, or an analytics `dashboard`.
+Later (in a real project), we would add other apps, such as an `admin` panel, a JSON `API`, or an analytics `dashboard`.
```

Comment:
```suggestion
Later (in a real project), we would add other apps, such as an `admin` panel, a JSON `api`, or an analytics `dashboard`.
```
I agree API should be capitalized, but this context is the name of the app, which _has_(?) to be lowercase.